build-support/integ_test_android.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+if [ ! -d android ]; then
+    echo "No Android project to test" >&2
+    exit
+fi
+
+TARGET=integration_test/main_test.dart
+if [ ! -e $TARGET ]; then
+    echo "$TARGET file not found" >&2
+    exit
+fi
+
+flutter test \
+    --no-pub \
+    -d sdk \
```

Comment:
question: Can the device argument be an optional argument passed to the script, with "sdk" as the default value? For some reason, in CI, "sdk" does not resolve and I've been passing "emulator-5554" instead, but I think it would help if this script accepted an argument for this.

Reply:
Updated the script to allow passing flag arguments.

Supported arguments see [here](https://github.com/aws-amplify/amplify-flutter/pull/1351/files#diff-c2b60e08277abc187525d4594d19b5d56fe925f1e710af6f08bdd99b2f0eab28R13-R21).